benchmarks/user/benchmark.py

```diff
@@ -56,12 +56,11 @@ def option_performance(f):
 
     _preset = {
         # Fixed
-        'O1': {'dse': 'basic', 'dle': 'basic'},
+        'O1': {'dse': 'basic', 'dle': 'advanced'},
```

Comment:
really?

Reply:
yes, I dropped dle basic, it was useless